packages/shared_preferences/example/lib/main.dart

```diff
@@ -29,13 +29,25 @@ class SharedPreferencesDemo extends StatefulWidget {
 }
 
 class SharedPreferencesDemoState extends State<SharedPreferencesDemo> {
-  Future<SharedPreferences> _prefs = SharedPreferences.getInstance();
```

Comment:
Let's keep this member var.
`shared_preferences` is typically a service in Flutter apps which is initialized at app init time. So, they don't access it this way. (I know in this particular sample app, it makes not much of a difference).